lib/github/oauth.js

```diff
@@ -63,7 +63,14 @@ module.exports = (robot) => {
     req.log.debug(accessTokenResponse.data, 'Exchanged code for access token');
     const accessToken = queryString.parse(accessTokenResponse.data).access_token;
 
-    const github = GitHubAPI({ logger: req.log });
+    const github = GitHubAPI({
+      Octokit: ProbotOctokit,
+      logger: req.log,
+      retry: {
+        // disable retries to test error states
+        enabled: false,
+      },
```

Comment:
I think there are a few more places where we need to disable throttling like here ☝️

Reply:
Thanks for pointing that out. Fixed in https://github.com/integrations/slack/pull/877/commits/809063711c94dbb1b4fe90ba994e6fd190167416